### PR TITLE
Add missing keys to the example secrets file

### PR DESCRIPTION
**Pull request by jfarrelly-voysis**
Also, correct the `aws-assume-role` key name

### CODE REVIEW

**Comment by jfarrelly-voysis** [comment]
Hi @stevesloka - Do I need to create an issue before creating a PR?  Are externals PRs still being accepted into this repository?

**Comment by stevesloka** [comment]
No @jfarrelly-voysis I've just been distracted, sorry for not getting back sooner. I'll take a quick look now.

File: k8s/secret.yaml [hunk 1]
```diff
@@ -27,9 +27,10 @@ metadata:
 data:
   AWS_ACCESS_KEY_ID: Y2hhbmdlbWU=
   AWS_SECRET_ACCESS_KEY: Y2hhbmdlbWU=
+  AWS_SESSION_TOKEN: Y2hhbmdlbWU=
   aws-account: Y2hhbmdlbWU=
-  aws-region: dXMtZWFzdC0x
-  aws_assume_role: YXJuOmF3czppYW06Ojk5OTk5OTk5OTpyb2xlL3JvbGUtYXJuLTIzNDIzNDIzZXRj
```
Comment (stevesloka) [inline k8s/secret.yaml]: I'm inconsistent with the underscore vs hyphen, the assume role is consumed with underscore's now: https://github.com/upmc-enterprises/registry-creds/blob/master/main.go#L70

Comment (stevesloka) [reply k8s/secret.yaml]: We could take a PR to standardize on hyphens, but changing to hyphens now would break the app.

Comment (jfarrelly-voysis) [reply k8s/secret.yaml]: Oh.  The replication controller kept crashing for me _until_ I converted the underscores to hyphens. Perhaps I have an older version of minikube (`v0.25.0`)

Comment (stevesloka) [reply k8s/secret.yaml]: Are you enabling via the add-on? Might be swapped on how we setup in minikube.

Comment (jfarrelly-voysis) [reply k8s/secret.yaml]: The replication controller in minikube seems to expect `aws-assume-role`:

https://github.com/kubernetes/minikube/blob/4d060d3bdba200f531fa00f04904e7076cb69c26/deploy/addons/registry-creds/registry-creds-rc.yaml#L57

Comment (jfarrelly-voysis) [reply k8s/secret.yaml]: Yes, after I load the secrets I do `minikube addons enable registry-creds`

**Pull request merged**